server/api/api/endpoints/background_tasks.py

```diff
@@ -199,14 +200,22 @@ async def _authorize_get_background_task_request(
     background_task: mlrun.common.schemas.BackgroundTask,
     auth_info: mlrun.common.schemas.AuthInfo,
 ):
+    return await asyncio.sleep(1)
```

Comment:
```suggestion
    return await asyncio.sleep(0)
```

but actually perhaps you can simply return without even sleep